NEWS.md

```diff
@@ -59,6 +59,8 @@
 
 * `across()` now defaults to `.cols = everything()` when `.cols` isn't provided (@markfairbanks, #231).
 
+* Can use `T` to specify the default in `case_when()`, (#272).
```

Comment:
Would you mind putting this in a separate PR?

Reply:
Yep, can do